blazor/ai-assistview/gemini-integration.md

```diff
@@ -0,0 +1,122 @@
+* Ensure you’re in the project root directory where your `.csproj` file is located.
+* Run the following command to install a [Syncfusion.Blazor.InteractiveChat](https://www.nuget.org/packages/Syncfusion.Blazor.InteractiveChat) and [Syncfusion.Blazor.Themes](https://www.nuget.org/packages/Syncfusion.Blazor.Themes/) NuGet package and ensure all dependencies are installed.
+
+{% tabs %}
+
+{% highlight c# tabtitle="Package Manager" %}
+
+dotnet add package Syncfusion.Blazor.InteractiveChat -v {{ site.releaseversion }}
+dotnet add package Syncfusion.Blazor.Themes -v {{ site.releaseversion }}
+dotnet restore
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> Syncfusion<sup style="font-size:70%">&reg;</sup> Blazor components are available in [nuget.org](https://www.nuget.org/packages?q=syncfusion.blazor). Refer to [NuGet packages](https://blazor.syncfusion.com/documentation/nuget-packages) topic for available NuGet packages list with component details.
+
+{% endtabcontent %}
+
+{% endtabcontents %}
+
+## Generate API Key
+
+1. Go to [Google AI Studio](https://aistudio.google.com/app/apikey) and sign in with your Google account. If you don’t have one, create a new account.
+
+2. Once logged in, click on `Get API Key` from the left-hand menu or the top-right corner of the dashboard.
+
+3. Click the `Create API Key` button. You’ll be prompted to either select an existing Google Cloud project or create a new one. Choose the appropriate option and proceed. 
+
+4. After selecting or creating a project, your API key will be generated and displayed. Copy the key and store it securely, as it will only be shown once.
+
+> `Security Note`: Never commit the API key to version control. Use environment variables or a secret manager for production.
+
+##  Integration Gemini AI with AI AssistView
+
+> Add your generated `API Key` at the line 
+
+```bash
+
+const string GeminiApiKey = 'Place your API key here'; 
+
+```
+
+{% tabs %}
+{% highlight razor %}
+
+<div class="aiassist-container" style="height: 350px; width: 650px;">
+    <SfAIAssistView ID="aiAssistView" PromptSuggestions="@promptSuggestions" PromptRequested="@OnPromptRequest">
+        <AssistViews>
+            <AssistView>
+                <BannerTemplate>
+                    <div class="banner-content">
+                        <div class="e-icons e-assistview-icon"></div>
+                        <h3>AI Assistance</h3>
+                        <i>To get started, provide input or choose a suggestion.</i>
+                    </div>
+                </BannerTemplate>
+            </AssistView>
+        </AssistViews>
+    </SfAIAssistView>
+</div>
+
+@code {
+    private List<string> promptSuggestions = new List<string>
+    {
+        "What are the best tools for organizing my tasks?",
+        "How can I maintain work-life balance effectively?"
+    };
+    private readonly string geminiApiKey = "AIzaSyB0AdTfrCZlkEaPFac8VoS55DUKfP5cyeE"; // Replace with your Gemini API key
```

Comment:
use same variable name casing as defined above @premanandh2205, remove the API key used. check other platforms.

<img width="461" height="67" alt="image" src="https://github.com/user-attachments/assets/afafb393-71a7-42cc-8d72-46a483934c33" />